syft/pkg/cataloger/debian/parse_deb_archive_test.go

```diff
@@ -0,0 +1,282 @@
+package debian
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"os"
+	"testing"
+	
+	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/pkg/cataloger/generic"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseDebArchive(t *testing.T) {
+	// Use the toilet package as a test fixture
+	fixture := "test-fixtures/toilet_0.3-1.4build1_amd64.deb"
```

Comment:
ideally we should try and not checkin binary test fixtures if we can help it. Can you change this to pull from a remote source or extract from a container image?